evennia/web/website/templates/website/_menu.html

```diff
@@ -40,8 +40,21 @@
             {% endblock %}
             {% block navbar_user %}
             {% if user.is_authenticated %}
-            <li class="nav-item">
-                <a class="nav-link">Logged in as {{user.username}}</a>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="user_options" aria-expanded="false">Logged in as {{user.username}} <span class="caret"></span></a>
+              <div class="dropdown-menu" aria-labelledby="user_options">
+                <a class="dropdown-item" href="{% url 'chargen' %}">Create</a>
```

Comment:
For clarity of intent, it would be better to call this menu item `New Character` or `Create Character`.